software/control/widgets.py

```diff
@@ -2319,26 +2319,30 @@ def enable_the_start_aquisition_button(self):
         self.btn_startAcquisition.setEnabled(True)
 
 
-class MultiPointWidget2(QFrame):
+class FlexibleMultiPointWidget(QFrame):
 
     signal_acquisition_started = Signal(bool)
     signal_acquisition_channels = Signal(list)
     signal_acquisition_z_levels = Signal(int)
-    signal_acquisition_shape = Signal(int, int, int, float, float, float)
+    signal_acquisition_shape = Signal(int, float)
```

Comment:
@sohamazing can you add in-line documentation here?

Reply:
added more here